File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/FloatPrimitiveListInPerson.java
package com.kinvey.androidTest.model;

import com.google.api.client.util.Key;

import java.util.List;

/**
 * Created by yuliya on 09/20/17.
 */

public class FloatPrimitiveListInPerson extends Person {

    @Key("floatList")
    private List<Float> floatList;

    public FloatPrimitiveListInPerson() {
    }

    public FloatPrimitiveListInPerson(String username) {
        this.username = username;
    }

    public List<Float> getFloatList() {
        return floatList;
    }

    public void setFloatList(List<Float> floatList) {
        this.floatList = floatList;
    }
}

File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/FloatPrimitiveListInPerson.kt
package com.kinvey.androidTest.model

import com.google.api.client.util.Key

/**
 * Created by yuliya on 09/20/17.
 */

class FloatPrimitiveListInPerson : Person {
    @Key("floatList")
    var floatList: List<Float>? = null

    constructor() {}
    constructor(username: String?) {
        this.username = username
    }

}